main.go

```diff
@@ -1,15 +1,32 @@
 package main
 
 import (
+	"database/sql"
+	"encoding/json"
 	"flag"
-	"log"
+	"fmt"
+
+	_ "github.com/lib/pq"
+
 	"net/http"
+	"strconv"
 	"strings"
 
 	"github.com/gorilla/websocket"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
```

Comment:
Postgresql has another format in which you can use a single long endpoint string to represent all this values. Use that one instead of moving around 5 different variables.